src/main/java/karuna/karuna_backend/receiver/infrastrucutre/ReceiverController.java

```diff
@@ -0,0 +1,100 @@
+package karuna.karuna_backend.receiver.infrastrucutre;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import karuna.karuna_backend.exception.dto.JwtErrorResponse;
+import karuna.karuna_backend.exception.dto.ReceiverErrorResponse;
+import karuna.karuna_backend.exception.dto.ValidationErrorResponse;
+import karuna.karuna_backend.receiver.domain.ReceiverService;
+import karuna.karuna_backend.receiver.dto.ReceiverCreateDto;
+import karuna.karuna_backend.receiver.dto.ReceiverDTO;
+import karuna.karuna_backend.receiver.dto.ReceiversDTO;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/receivers")
+class ReceiverController {
+
+    private final ReceiverService receiverService;
+
+    @GetMapping
+    @Operation(summary = "Get all recipients of visitor message", description = "Retrieves all recipients from database.")
+    @ApiResponses({
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "All recipients from database",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ReceiversDTO.class))),
+            @ApiResponse(
+                    responseCode = "403",
+                    description = "Forbidden: You don't have permission to access this resource"),
+            @ApiResponse(
```

Comment:
We should have every possibly request type so FE knows what to handle on their side. Since this 401 - 403 applies to every endpoint in this controller we might move this to GlobalReceiverException maybe to remove redundancy.